src/main/java/com/plugish/woominecraft/WooCommand.java

```diff
@@ -157,13 +157,18 @@ private void pingSubcommand(CommandSender sender) {
                 WooMinecraft.instance.getLogger().severe(e.getMessage());
                 sender.sendMessage(chatPrefix +ChatColor.DARK_RED+"Server Status: Failed");
                 if (plugin.isDebug()) {
-                    WooMinecraft.instance.getLogger().info(plugin.getConfig().getString("key"));
+                    //Do not output store keys to console, can open up potential issues
                     WooMinecraft.instance.getLogger().info(plugin.getConfig().getString("url"));
                 }
             }
         });
     }
     private void debugSubcommand(CommandSender sender) {
+        if (!sender.hasPermission("woo.admin")) {
+            String msg = chatPrefix + ChatColor.translateAlternateColorCodes('&', plugin.getLang("general.not_authorized"));
+            sender.sendMessage(msg);

```

Comment:
Just simplify this, no need for a vairable:
`sender.sendMessage( chatPrefix + ChatColor.translateAlternateColorCodes( '&', plugin.getLang( "general.not_authorized" ) ) );`